doc/wiki/2_KSS_RSI.md

```diff
@@ -119,41 +144,86 @@ Both launch files support the following arguments:
+
+##### Stopping external control
 
 To stop external control, all components have to be deactivated with `ros2 lifecycle set robot_manager deactivate`
 
 BEWARE, that this is a non-realtime process including lifecycle management, so the connection is not terminated immediately, in cases where an abrupt stop is needed, the safety stop of the Teach Pendant should be used!
 
-
-### Simulation
+#### Simulation
 
 To try out the driver with an open-loop simulation, the driver and the `kuka_rsi_simulator` must be started, (before activation only a "collapsed" robot will be visible in `rviz`):
 
-```
+```bash
 ros2 launch kuka_rsi_driver startup_with_rviz.launch.py
 ```
 
-```
+```bash
 ros2 launch kuka_rsi_simulator kuka_rsi_simulator_launch.py
 ```
 
 After all components have started successfully, the system needs to be configured and activated to start the simulation. The robot will be visible in rviz after activation:
 
-```
+```bash
 ros2 lifecycle set robot_manager configure
 ros2 lifecycle set robot_manager activate
 ```
 
-### Known issues and limitations
+#### Known issues and limitations
 
-- There are currently heap allocations in the control loop (hardware interface `read()` and `write()` functions), therefore the driver is not real-time safe
 - In case of an error on the controller side, the driver is not deactivated
 - Cartesian position control mode and I/O-s not yet supported
+
+### EKI + RSI driver
+
+This section explains how to configure and use the driver when both EKI and RSI are available on the KUKA controller.
+
+#### Setting up the EKI + RSI driver
+
+To set up the driver for use with both EKI and RSI, follow the instructions in the guide posted in the SDK repository: [External control setup for KSS with EKI](https://github.com/kroshu/kuka-external-control-sdk/blob/master/kuka_external_control_sdk/doc/kss_eki_setup.md)
+
+#### Launching the EKI + RSI driver
+
+To start the KUKA RSI driver, use one of the provided launch files:
+
+- `startup.launch.py`: Launches the driver only
+- `startup_with_rviz.launch.py`: Launches the driver along with RViz for visualization
+
+Regardless of the launch file you choose, the following arguments must be specified:
+
+- `robot_family`: Defines the family of the robot (e.g., agilus)
+- `robot_model`: Specifies the exact robot model. A list of supported models and their corresponding families is available in the `kuka_robot_descriptions` [README](https://github.com/kroshu/kuka_robot_descriptions?tab=readme-ov-file#what-data-is-verified)
+- `controller_ip`: The IP address of the **KUKA Line Interface (KLI)**
+- `driver_version`: Set this parameter to `eki_rsi`. If not set the plain RSI driver will be used.
+- `verify_robot_model`: If set to `true`, the driver will verify that the robot model the launch arguments match the configuration reported by the controller.
```

Comment:
should not we make this optional and describe in the additional launch arguments section?